### PR TITLE
Merge/slideshow jan2016

### PR DESCRIPTION
**Pull request by dereksmart**
Merge changes from wpcom

cc @lancewillett

### CODE REVIEW

**Comment by lancewillett** [comment]
Looks good to me. :+1:

File: modules/shortcodes/slideshow.php [hunk 1]
```diff
@@ -286,14 +306,16 @@ function enqueue_scripts() {
...
+			 * @since  2.1.0
 			 *
 			 * @param array $args
 			 * - string - spinner - URL of the spinner image.
 			 */
-			apply_filters( 'jetpack_js_slideshow_settings', array(
-				'spinner' => plugins_url( '/img/slideshow-loader.gif', __FILE__ ),
-			) )
+			apply_filters(
+				'jetpack_js_slideshow_settings', array(
```
Comment (kraftbj) [inline modules/shortcodes/slideshow.php]: will having `apply_filters` on multiple lines break the upcoming developer site @dereksmart?

File: modules/shortcodes/slideshow.php [hunk 2]
```diff
@@ -286,14 +306,16 @@ function enqueue_scripts() {
 			 *
 			 * @module shortcodes
 			 *
-			 * @since 2.1.0
+			 * @since  2.1.0
```
Comment (kraftbj) [inline modules/shortcodes/slideshow.php]: unneeded space

File: modules/shortcodes/slideshow.php [hunk 3]
```diff
@@ -137,39 +146,41 @@ function shortcode_callback( $attr, $content = null ) {
...
 			 * Filters the Slideshow slide caption.
 			 *
 			 * @module shortcodes
 			 *
-			 * @since 2.3.0
+			 * @since  2.3.0
 			 *
-			 * @param string wptexturize( strip_tags( $attachment->post_excerpt ) ) Post excerpt.
-			 * @param string $attachment->ID Attachment ID.
+			 * @param        string      wptexturize( strip_tags( $attachment->post_excerpt ) ) Post excerpt.
```
Comment (kraftbj) [inline modules/shortcodes/slideshow.php]: spacing

File: modules/shortcodes/slideshow.php [hunk 4]
```diff
@@ -137,39 +146,41 @@ function shortcode_callback( $attr, $content = null ) {
...
 			$attachment_image_title = get_the_title( $attachment->ID );
-			$attachment_image_alt = get_post_meta( $attachment->ID, '_wp_attachment_image_alt', true );
+			$attachment_image_alt   = get_post_meta( $attachment->ID, '_wp_attachment_image_alt', true );
 			/**
 			 * Filters the Slideshow slide caption.
 			 *
 			 * @module shortcodes
 			 *
-			 * @since 2.3.0
+			 * @since  2.3.0
```
Comment (kraftbj) [inline modules/shortcodes/slideshow.php]: spacing

File: modules/shortcodes/slideshow.php (hunk 1 above)
Comment (dereksmart) [inline modules/shortcodes/slideshow.php]: shouldn't be a problem.

**Pull request merged**